### PR TITLE
Payment gateways

### PR DESCRIPTION
**Pull request by kuiche**
Store the gateway against the payment.

### CODE REVIEW

**Comment by thomasjthomasj** [comment]
Docblocks plz

**Comment by kuiche** [comment]
Keen ;)

File: src/Payment/PaymentGatewayRecordLoader.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+/**
+ * @author Samuel Trangmar-Keates <sam@message.co.uk>
+ *
+ * Loads payment-gateway records. Either the gateway for an payment or all payments
+ * created using a gateway.
+ */
+class PaymentGatewayRecordLoader
+{
+	private $_queryBuilderFactory;
```
Comment (thomasjthomasj) [inline src/Payment/PaymentGatewayRecordLoader.php]: Please add docblocks for these properties

File: src/Payment/PaymentGatewayRecordEdit.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+use Message\Cog\DB\Query;
+
+/**
+ * @author Samuel Trangmar-Keates <sam@message.co.uk>
+ *
+ * Class for saving and updating payment-gateway mapping records
+ */
+class PaymentGatewayRecordEdit
+{
+	private $_query;
```
Comment (thomasjthomasj) [inline src/Payment/PaymentGatewayRecordEdit.php]: Docblock for this please

**Pull request merged**